src/Compatibility/Microsoft.DotNet.ApiCompatibility/ApiComparer.cs

```diff
@@ -37,6 +37,11 @@ public class ApiComparer
         /// </summary>
         public bool StrictMode { get; set; }
 
+        /// <summary>
+        /// Flag indicating whether the ApiComparison will be performed using references or not.
+        /// </summary>
+        public bool RunningWithReferences { get; set; }
```

Comment:
Seems like RunningWithReferences is both here and in ComparingSettings, so perhaps we should remove them from here